src/kaleidoscope/Hardware.h

```diff
@@ -385,6 +391,20 @@ class Hardware {
    */
   void enableHardwareTestMode() {}
 
+  /**
+   * Method to return the object the hardware uses for storage.
+   */
+  auto storage() -> decltype(EEPROM) {
```

Comment:
Is this meant to return by value? If not, the signature must read
```cpp
auto storage() -> decltype(EEPROM) & {
```

Reply:
Aha, so that's how you do a reference with auto! Fixed, thanks.